### PR TITLE
Streamline generic image

### PR DESCRIPTION
**Pull request by qdrop17**
Remove browsers and Microsoft Teams from image.

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -10,27 +10,13 @@ RUN yes | unminimize > /dev/null 2>&1
...
-Pin-Priority: 1001' | tee /etc/apt/preferences.d/mozilla-firefox
-
-RUN echo 'Unattended-Upgrade::Allowed-Origins:: "LP-PPA-mozillateam:${distro_codename}";' | tee /etc/apt/apt.conf.d/51unattended-upgrades-firefox
-RUN apt-get -qq update && apt-get -qq install -y firefox > /dev/null 2>&1
-
+apt-get -qq update && apt-get -qq install -y sudo vim syslog-ng zram-config earlyoom iputils-ping dnsutils traceroute ca-certificates conky openssh-server libxss1 > /dev/null 2>&1'
 
 # Disable pop up on first opening of chrome
-RUN sed -i 's,Exec=/usr/bin/google-chrome-stable,Exec=/usr/bin/google-chrome-stable --no-first-run --no-default-browser-check,g' /usr/share/applications/google-chrome.desktop
+# RUN sed -i 's,Exec=/usr/bin/google-chrome-stable,Exec=/usr/bin/google-chrome-stable --no-first-run --no-default-browser-check,g' /usr/share/applications/google-chrome.desktop
```
Comment (mazlumtoprak) [inline build/Dockerfile]: can be removed, if it won't be executed

Comment (qdrop17) [reply build/Dockerfile]: is gone...

File: build/Dockerfile [hunk 2]
```diff
@@ -10,27 +10,13 @@ RUN yes | unminimize > /dev/null 2>&1
...
-
-RUN echo 'Unattended-Upgrade::Allowed-Origins:: "LP-PPA-mozillateam:${distro_codename}";' | tee /etc/apt/apt.conf.d/51unattended-upgrades-firefox
-RUN apt-get -qq update && apt-get -qq install -y firefox > /dev/null 2>&1
-
+apt-get -qq update && apt-get -qq install -y sudo vim syslog-ng earlyoom iputils-ping dnsutils traceroute ca-certificates conky openssh-server libxss1 > /dev/null 2>&1'
 
-# Disable pop up on first opening of chrome
-RUN sed -i 's,Exec=/usr/bin/google-chrome-stable,Exec=/usr/bin/google-chrome-stable --no-first-run --no-default-browser-check,g' /usr/share/applications/google-chrome.desktop
+# Install Intel proprietary drivers
+RUN bash -c 'kernelVersion=$(curl -s http://$NETBOOT_IP/kernels/latest-kernel-version.json | jq -r .version) && apt-get -qq update && apt-get -qq install -y linux-modules-extra-$kernelVersion linux-firmware > /dev/null 2>&1'
```
Comment (mazlumtoprak) [inline build/Dockerfile:19]: would make sense to integrate the package installations on the RUN command from line 15. $kernelVersion should be already set there.

File: build/Dockerfile [hunk 3]
```diff
@@ -10,27 +10,13 @@ RUN yes | unminimize > /dev/null 2>&1
...
-# Install firefox without snap on ubuntu-22.04
-RUN add-apt-repository ppa:mozillateam/ppa -y && \
-    echo 'Package: * \n\
-Pin: release o=LP-PPA-mozillateam \n\
-Pin-Priority: 1001' | tee /etc/apt/preferences.d/mozilla-firefox
-
-RUN echo 'Unattended-Upgrade::Allowed-Origins:: "LP-PPA-mozillateam:${distro_codename}";' | tee /etc/apt/apt.conf.d/51unattended-upgrades-firefox
-RUN apt-get -qq update && apt-get -qq install -y firefox > /dev/null 2>&1
-
+apt-get -qq update && apt-get -qq install -y sudo vim syslog-ng earlyoom iputils-ping dnsutils traceroute ca-certificates conky openssh-server libxss1 > /dev/null 2>&1'
```
Comment (mazlumtoprak) [inline build/Dockerfile:16]:
```suggestion
apt-get -qq update && apt-get -qq install -y sudo vim syslog-ng earlyoom iputils-ping dnsutils traceroute ca-certificates conky openssh-server libxss1 linux-modules-extra-$kernelVersion linux-firmware > /dev/null 2>&1'
```

**Review by ccadruvi — APPROVED** [review]

**Pull request merged**